src/testTree.ts

```diff
@@ -251,13 +241,6 @@ export class TestTree {
     return result || fileItem;
   }
 
-  private _tagFileItem(fileItem: vscodeTypes.TestItem, projectTag: vscodeTypes.TestTag) {
```

Comment:
IIUC, we don't need tags anymore, because the user can only run tests from enabled projects==configurations now anyway.